docs/docs/site-showcase-submissions.md

```diff
@@ -17,10 +17,10 @@ There are only three major steps :)
 ```yaml
 - title: Title of the Site
 
-  # this URL is used to generate a screenshot
+  # this is the URL that is linked from the showcase
   main_url: https://titleofthesite.com
 
-  # this is the URL that is linked from the showcase
+  # this URL is used to generate a screenshot
```

Comment:
Oops, thanks! Maybe we should rename these to `screenshot_url` and `link_url` or similar.